src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/messenger_multiple_buses.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+$container->loadFromExtension('framework', array(
+    'messenger' => array(
+        'default_bus' => 'commands',
+        'buses' => array(
+            'commands' => null,
+            'events' => array(
+                'middlewares' => array(
+                    'Symfony\\Component\\Messenger\\Middleware\\TolerateNoHandler',
+                ),
+            ),
+            'queries' => array(
+                'default_middlewares' => false,
+                'middlewares' => array(
+                    'route_messages',
+                    'Symfony\\Component\\Messenger\\Middleware\\TolerateNoHandler',
```

Comment:
It's a bit misleading reading the tests and seing this FQCN while the actual middleware exists and has a service id that should be used (or the shortcut), not the FQCN. Is it really relevant to test a FQCN here?

Reply:
Good point; changed.